### ISSUE TITLE
Add a way to perform batch inserts

### ISSUE THREAD

**Issue by ExplodingCabbage** [Daursu/xero#9]
The Xero API allows batch inserts by doing a PUT or POST request with a body like

``` xml
<?xml version='1.0'?>
<FooBars>
    <FooBar>
        <Stuff>bla</Stuff>
    <FooBar>
    <FooBar>
        <Stuff>moo</Stuff>
    <FooBar>
</FooBars>
```

These batch uploads are important for lots of people (me included) to work around the [API limits](http://developer.xero.com/documentation/getting-started/xero-api-limits/).

As far as I can tell, there's no (non-hacky) way to perform them with this library at present.

**Comment by ExplodingCabbage** [comment]
Perhaps collections should offer a `->save()` method?

**Comment by Daursu** [comment]
Yes a `save()` method on the collection should do the trick. I will have a look at implementing it.
